electron/services.js

```diff
@@ -109,25 +111,64 @@ export class Services {
         return Promise.all(shuttingDown);
     }
 
+    tryExistingRpc() {
+        return new Promise((resolve, reject) => {
+            check("http://localhost:8545").then((status) => {
+                if (!status.exists) {
+                    reject(new Error("Not exists"));
+                    return;
+                }
+                if (!this.setup.chain) {
+                    this.setup.chain = status.chain;
+                    resolve('setup from existing')
+                } else if (status.chain === this.setup.chain) {
+                    resolve('connect to existing')
+                } else {
+                    reject(new Error(`Wrong chain ${this.setup.chain} != ${status.chain}`))
```

Comment:
yeah, I think it's better to just switch settings to existing chain, as we can't stop existing geth and can't run another instance on same port (we can run on another port, but it adds additional complication). I'll update PR